### PR TITLE
Organisation summury list.  Populate your detail page from session.

### PR DESCRIPTION
**Pull request by dpatel017**
Organisation summury list.  Populate your detail page from session.

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml.cs [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
 
-    public OrganisationDetailsSummaryModel(ILogger<OrganisationDetailsSummaryModel> logger)
+    public void OnGet()
     {
-        _logger = logger;
+        RegistrationDetailModel = VerifySession();
     }
 
-    public void OnGet()
+    private RegistrationDetails VerifySession()
```
Comment (glgoaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml.cs:16]: Could the session access be abstracted into a separate service?

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml [hunk 2]
```diff
@@ -1,72 +1,119 @@
...
 
             <fieldset class="govuk-fieldset">
                 <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                    <h1 class="govuk-fieldset__heading">
-                        Confirm your organisation's details
+                    <h1 class="govuk-heading-l">
+                        <span class="govuk-label--l">Overview</span>
                     </h1>
+
+                    <h2 class="govuk-heading-m">Organisation details</h2>
```
Comment (glgoaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml:23]: We potentially could create a summary list partial view

**Review by glgoaco — APPROVED** [review]
LGTM

**Review by JBaigGoaco — APPROVED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml (hunk 2 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml:23]: Yes, we can refactor it.  Let's have dev discussion on it.

**Pull request merged**